scripts/zkai_attention_kv_proof_route_selector_gate.py

```
@@ -535,6 +595,28 @@ def load_stwo_native_masked_sequence_envelope(
     return envelope
 
 
+@functools.lru_cache(maxsize=None)
```

Ruff (0.15.12)

[warning] Use `@functools.cache` instead of `@functools.lru_cache(maxsize=None)`

Rewrite with `@functools.cache

(UP033)